src/main/java/ca/corefacility/bioinformatics/irida/model/enums/StatisticTimePeriod.java

```diff
@@ -0,0 +1,25 @@
+package ca.corefacility.bioinformatics.irida.model.enums;
+
+public enum StatisticTimePeriod {
```

Comment:
Comments

Reply:
Added in 06bf685f3158a23f35de733bead1341113532cd5